neatformcore/src/main/java/com/nerdstone/neatformcore/utils/ViewUtils.kt

```diff
@@ -25,57 +25,110 @@ import java.util.*
 object ViewUtils {
 
     fun createViews(
-        rootView: RootView, viewProperties: List<NFormViewProperty>,
-        context: Context, viewDispatcher: ViewDispatcher
+            rootView: RootView, viewProperties: List<NFormViewProperty>,
+            context: Context, viewDispatcher: ViewDispatcher
     ) {
 
         for (viewProperty in viewProperties) {
             when (viewProperty.type) {
                 ViewType.EDIT_TEXT ->
                     rootView.addChild(
-                        getView(EditTextNFormView(context), viewProperty, viewDispatcher)
+                            getView(EditTextNFormView(context), viewProperty, viewDispatcher)
                     )
                 ViewType.MULTI_CHOICE_CHECKBOX ->
                     rootView.addChild(
-                        getView(MultiChoiceCheckBox(context), viewProperty, viewDispatcher)
+                            getView(MultiChoiceCheckBox(context), viewProperty, viewDispatcher)
                     )
                 ViewType.CHECKBOX ->
                     rootView.addChild(
-                        getView(CheckBoxNFormView(context), viewProperty, viewDispatcher)
+                            getView(CheckBoxNFormView(context), viewProperty, viewDispatcher)
                     )
                 ViewType.SPINNER ->
                     rootView.addChild(
-                        getView(SpinnerNFormView(context), viewProperty, viewDispatcher)
+                            getView(SpinnerNFormView(context), viewProperty, viewDispatcher)
                     )
                 ViewType.RADIO_GROUP ->
                     rootView.addChild(
-                        getView(RadioGroupView(context), viewProperty, viewDispatcher)
+                            getView(RadioGroupView(context), viewProperty, viewDispatcher)
                     )
                 ViewType.DATETIME_PICKER ->
                     rootView.addChild(
-                        getView(DateTimePickerNFormView(context), viewProperty, viewDispatcher)
+                            getView(DateTimePickerNFormView(context), viewProperty, viewDispatcher)
                     )
                 ViewType.NUMBER_SELECTOR ->
                     rootView.addChild(
-                        getView(NumberSelectorNFormView(context), viewProperty, viewDispatcher)
+                            getView(NumberSelectorNFormView(context), viewProperty, viewDispatcher)
                     )
+                ViewType.TEXT_INPUT_LAYOUT ->
+                    rootView.addChild(
+                            getView(TextInputLayoutNFormView(context), viewProperty, viewDispatcher)
+                    )
+            }
+
+        }
+    }
+
+    fun updateViews(
```

Comment:
We can just have one method and just pass a flag to differentiate **update** or **create** so that we only maintain one function that handles both cases